chemprop/v2/tests/unit/featurizers/test_bond.py

```diff
@@ -42,7 +42,7 @@ def bt_bit(bond, bond_types, featurizer):
     if i == -1:
         return i
 
-    return range(len(featurizer))[featurizer.subfeatures["bond_type"].start + i]
+    return featurizer.one_hot_index(bt, featurizer.bond_types)[0] + 1 
```

Comment:
Why do we + 1 here? Also could you remove the extra blank at the end of line

Reply:
The first element in a bond feature is "if this bond is `None`". Index 1 to len(bond_types) are related to bond types.
 `one_hot_index()` returns the index of `bt` in the list of `bond_types`, so you need to add `1` to obtain the actual bond type bit in the bond feature.

Reply:
I removed the extra blank.